### DISCUSSION TITLE
当文本文件太长时 手机端需要翻到最底下才能下载

### DISCUSSION THREAD

**Discussion by wukibaka** [spencerwooo/onedrive-vercel-index#198] [Ideas]
一些小白不会切换ua 存的一些文本需要分享给他 每次他都要翻很久才能翻到最下面找到下载按钮

**Comment by myl7** [comment]
或许可以把按钮挪到上方，或者是加一个配置项允许把按钮挪到上方？

**Reply by wukibaka** [reply]
可以的话，给文本预览加个滚动条？

**Comment by spencerwooo** [comment]
https://user-images.githubusercontent.com/32114380/147640326-2d571c67-27d5-446b-867c-8def248f8759.mp4

老板还满意吗？🥳

**Reply by wukibaka** [reply]
好耶！ 大佬万岁！
